src/components/App/App.css

```diff
@@ -1,70 +1,142 @@
 main {
+  background-color: black;
   height: 100vh;
   width: 100vw;
 }
 
-h1 {
-  color: #ad9976;
-}
-
 .window {
   display: grid;
-  height: 90vh;
+  height: 85%;
   grid-template-rows: 18% 7% 12% 20% 11% 9% 24%;
   grid-template-columns: 6% 13% 20% 24% 18% 15% 7%;
-  width: 80vw;
+  margin-left: 5%;
+  margin-right: 5%;
+  /* width: 88%; */
 }
 
 .preview-pane {
   background-color: red;
-  grid-column: 2 / span 4;
+  clip-path: polygon(7% 44%, 69% 5%, 74% 67%, 15% 99%);
+  grid-column: 1 / span 3;
   grid-row: 1 / span 2;
 }
 
+.preview-pane > img {
+  color: white;
+  height: 5rem;
+  width: 5rem;
+  position: absolute;
+  top: 10rem;
+  left: 2rem;
+}
+
 .countdown-pane {
   background-color: aqua;
-  grid-column: 4 / span 5;
+  clip-path: polygon(27% 38%, 83% 0%, 81% 76%, 32% 100%);
```

Comment:
Is there a way to use clip path with a pentagon shape to get the domed pane we were looking for?

Reply:
It will unfortunately just give us more sharp corners instead of a rounded top. I'm thinking we can put a transparent circle with a black border to create the appearance of a rounded top